src/main/java/mServer/crawler/AddToFilmlist.java

```diff
@@ -5,176 +5,270 @@
  */
 package mServer.crawler;
 
+import java.io.IOException;
+import java.net.SocketTimeoutException;
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.HashSet;
-import java.util.Iterator;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import etm.core.configuration.EtmManager;
-import etm.core.monitor.EtmMonitor;
-import etm.core.monitor.EtmPoint;
-import mSearch.Config;
-import mSearch.daten.DatenFilm;
-import mSearch.daten.ListeFilme;
-import mSearch.tool.Duration;
-import mSearch.tool.FileSize;
-import mSearch.tool.Log;
-
-/**
- *
- * @author emil
- */
+import de.mediathekview.mlib.Config;
+import de.mediathekview.mlib.daten.DatenFilm;
+import de.mediathekview.mlib.daten.ListeFilme;
+import de.mediathekview.mlib.tool.Hash;
+import de.mediathekview.mlib.tool.Log;
+import de.mediathekview.mlib.tool.MVHttpClient;
+import mServer.tool.MserverDaten;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
 public class AddToFilmlist {
-    private static final EtmMonitor etmMonitor = EtmManager.getEtmMonitor();
-    AtomicInteger threadCounter = new AtomicInteger(0);
-    AtomicInteger treffer = new AtomicInteger(0);
-    ListeFilme vonListe;
-    ListeFilme listeEinsortieren;
-    Collection<DatenFilm> filteredOnline = new ArrayList<>();
+    /**
+     * Minimum size of films in MiB to be included in new list.
+     */
+    private static final int MIN_SIZE_ADD_OLD = 5;
+    private final static int NUMBER_OF_THREADS = 32;//(Runtime.getRuntime().availableProcessors() * Runtime.getRuntime().availableProcessors()) / 2;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)